docs/ha_3scale.adoc

```diff
@@ -217,4 +217,67 @@ In this setup, to ensure good database access latency, each cluster will have it
 . Manual failovers of the databases to execute are (order does not matter):
 .. AWS RDS: *System* and *Zync*
 .. AWS ElastiCaches: *Backend* and *System*
-. Execute step 4 and 5 from <<manual-failover-shared-databases>>
\ No newline at end of file
+. Execute step 4 and 5 from <<manual-failover-shared-databases>>
+
+=== Active-Active clusters (not supported)
+
+Having an application working on *active/active* configuration is always difficult for any application:
+
+- *Soft Databases limitation*: it is complex but can be achieved with some trade offs, but it is mainly responsibility from the administrator (not 3scale responsibility)
+- *Hard Application limitation*: this one is a hard limitation imposed by how 3scale manages OpenShift Routes and cannot be achieved
+
+==== Soft Databases limitation (out of 3scale scope)
+
+One of the most difficult parts for any application deployed on *active*/*active* (not only 3scale), is having databases on *active*/*active* mode, and in case of 3scale there are many different databases involved which would require different *active*/*active* implementations for each of them:
+
+- System-mysql (or system-postgres or system-oracle, you can choose)
+- System-redis
+- Backend-redis
+- Zync-database (postgresql)
+- system-sphinx (doesn't need HA)
+- system-memcached (doesn't need HA)
+
+So, from 3scale point of view, an administrator need to ensure that database connection strings configured in 3scale APIManager custom resource on any OCP cluster will be always a master instance (allowing write operations).
+
+____
+*NOTE*
+
+*This database limitation is a soft limitation because although being very complex (implementing active/active for mysql, postgresql or redis is not trivial), it can be achieved.*
+____
+
+==== Hard Application limitation: OpenShift Routes
```

Comment:
is this the only hard limitation we have in the whole 3scale logic that makes supporting Active/Active difficult (ignoring the DB active/active complexities for now)?

Reply:
This is the only one I thought, but everything  written here is theoretical, that's why I preferred to ask you in case you might think about other possible limitations knowing the things you have seen in support team done by on-prem customers